src/graphql/resolvers.ts

```diff
@@ -0,0 +1,28 @@
+import appConfig from '../../app.config'
+import {
+  AggregationResult,
+  formatGraphQLResults,
+  getResults
+} from '@components/Search/utils'
+
+export const getChainIds = async (): Promise<number[]> => {
+  const { chainIds } = appConfig
+  return chainIds.filter((id) => id !== 3 && id !== 4)
```

Comment:
what are these magic numbers? can we unmagic them?

Reply:
Fixed the syntax